dapps/assets/src/components/contacts/AddContact.ts

```diff
@@ -21,7 +21,7 @@
 import Component, { mixins } from 'vue-class-component';
 
 // evan.network imports
-import { EvanComponent, EvanForm } from '@evan.network/ui-vue-core';
+import { EvanComponent, EvanForm, SwipePanelComponentClass } from '@evan.network/ui-vue-core';
```

Comment:
this is probably bit out of scope, but what is actually the reason, that we use here an extra SwipePanelComponentClass instead of SwipePanelComponent?

Reply:
It's a bit tricky with typescript definitions and vue files. The `SwipePanel` itself represents the vue component, that can be registered and referenced to be added within the template. The `SwipePanelComponentClass` is the typescript class definition of this component, without the vue stuff, so typings can be used correctly, without a `any` call.

Reply:
Somehow the `SwipePanelComponent` doesn't include the functions. I haven't looked into this myself tbh.